js/scripts.js

```diff
@@ -199,3 +192,22 @@ window.onclick = function clickOut(event) {
     clearModal();
   }
 };
+
+function haveUpper(str) {
+  return /[A-Z]/.test(str);
+}
+
+document.querySelector('form').addEventListener('submit', (event) => {
+  const formEmail = document.getElementById('form-email');
+  const alert = document.querySelector('.alert');
+  if (haveUpper(formEmail.value)) {
+    alert.innerHTML = 'Your e-mail needs to be in lowercase';
```

Comment:
- [ ] As per project requirement The error message fits the design from the chosen template in Figma (e.g., font, colours, images, tests, margins). But right now it is not following the design guidelines. Kindly try to change the font size, Background colour and colour to adhere to the given design.

![image](https://user-images.githubusercontent.com/53019916/131799301-ea7e558d-d671-4171-8798-606afb14b6aa.png)

Reply:
Hi @hemant-soni-vst-au4 , can you show me where I can find the error styles from Figma (Template 1), because I didn't find any reference for them.

Reply:
- [ ] Hello @lfmnovaes,
The error style is not present in the Figma template. But as per requirements, we should make the error style similar to the design pattern. Like we can change the background colour similar to the colour of our Figma design theme.

Reply:
- [ ] Correct buddy, For error message also we have to follow the colour, text, as per Figma design.